### PR TITLE
chore: all-crews end-to-end verification sweep (15/15 green)

### PR DESCRIPTION
**Pull request by fjacquet**
## What

Applies the holiday flow's reliability treatment to **every** routable `ReceptionFlow`
crew: runs all 14 remaining flows end-to-end on the current stack
(native Gemini `gemini/gemini-3.5-flash` + forced ReAct tool-calling) and fixes every
latent output/render/email failure that surfaced. **15/15 flows now green** — valid
report generated + email delivered to the test inbox, error log empty.

## Fixes (each a distinct runtime failure the sweep surfaced)

- **`fede645` fix(flow):** `kickoff()` returns `None` so a successful run exits 0. It was
  returning the flow object, which `sys.exit()` printed and exited 1 — every successful
  run looked like a failure. **Global** (affected all flows).
- **`f57c5e9` fix(rss):** `generate_rss_weekly` awaits `akickoff_flow` (was calling the
  sync `kickoff_flow` from an async method).
- **`b70bde0` fix(menu):** `generate_menu_designer` sets `state.output_file` so the menu
  report is the emailed attachment (it was stuck on the classify path — report generated
  but never emailed).

## Tooling & docs

- **`eebe6bb` / `297ebd7`:** `kickoff()` honors an `EPIC_NEWS_REQUEST` env override, plus
  `scripts/verify_all_crews.sh` — a repeatable driver that runs one flow or all via
  `EPIC_NEWS_REQUEST` + `MAIL`. Sentinel workflow in `main.py` is unchanged (env falls back
  to the hardcoded query when unset).
- **`4a4df8c` fix(sweep):** driver now fails loudly (exit 2) on an unknown flow label
  instead of silently running nothing and exiting 0.
- **`5d97afd` docs:** `docs/sweep-runbook.md` — how to run, how to read results (incl. the
  `email_delivered` cumulative-log false-positive), the 15/15 results table, and the soft notes.

## Results

| Flow | Result |
|---|---|
| SAINT, COOKING, SHOPPING, BOOK_SUMMARY, NEWSDAILY, FINDAILY, COMPANY_NEWS | PASS |
| RSS | PASS (after `f57c5e9`) |
| MENU | PASS (after `b70bde0`) |
| MEETING_PREP, PESTEL, SALES_PROSPECTING, DEEPRESEARCH, OPEN_SOURCE_INTELLIGENCE | PASS |
| HOLIDAY_PLANNER | PASS (verified earlier, PR #158) |

POEM excluded per decision.

### Non-blocking soft notes (external / self-healing, not fixed)

- PESTEL: transient ReAct "Action Input is not a valid key, value dictionary" parser
  retries — recovered, run completed clean.
- SALES_PROSPECTING: agents probed non-existent scratchpad `.txt` files — recovered, no
  error leaked into the report.
- DEEPRESEARCH: third-party Wikipedia MCP `search` returned an external `403` (Wikipedia
  API rate-limit) — agent fell back to other tools.

## Test plan

- `uv run pytest` → **729 passed, 5 skipped**.
- All 15 flows run end-to-end via `scripts/verify_all_crews.sh`; each delivered a real
  email to `fred.jacquet@gmail.com` with the correct report attached; `epic_news_error.log`
  empty after every run.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_0174YSPJ8Uc2wtheHmd56vaE

## Summary by CodeRabbit

* **New Features**
  * Added an all-crews end-to-end verification driver to run and review workflow results.
  * Introduced environment-based request routing and optional test-inbox delivery markers.
  * Added loud failure and diagnostics for unknown workflow labels.
* **Bug Fixes**
  * Improved async RSS translation execution behavior.
  * Fixed weekly menu email to use the final rendered report as the attachment source.
  * Improved clean exits for workflow kickoff.
  * Hardened cooking HTML rendering for multiline ingredients and numbered instructions.
* **Documentation**
  * Added verification sweep plans and an operational runbook.
* **Tests**
  * Added coverage for request override behavior and cooking HTML rendering edge cases.

### CODE REVIEW

File: src/epic_news/main.py [hunk 1]
```diff
@@ -1501,8 +1508,12 @@ def kickoff(user_input: str | None = None):
     It can optionally take a user_input string to override the default.
 
     Returns:
-        The completed ReceptionFlow object.
+        None. The flow runs for its side effects; the console entry point runs
+        ``sys.exit(kickoff())``, which needs None/int — not the flow object.
     """
+    # Sweep/automation hook: let EPIC_NEWS_REQUEST drive the request without
+    # editing the hardcoded query below. An explicit user_input arg still wins.
+    user_input = user_input or os.getenv("EPIC_NEWS_REQUEST") or None
```
[the remark this answers is not in the document]
Comment (fjacquet) [reply src/epic_news/main.py]: Fixed in 7b1bc13. The override now only fires when no explicit `user_input` is passed, and emits a `logger.warning` (after `setup_logging`) naming the forced request, so a leaked/stale `EPIC_NEWS_REQUEST` can't silently reroute a real request without a trace in the logs.
